NAS2D/Renderer/Fade.cpp

```diff
@@ -88,7 +88,7 @@ void Fade::draw(Renderer& renderer) const
 
 void Fade::setDuration(const std::chrono::milliseconds& newDuration)
 {
-	if (newDuration.count() <= 0LL)
+	if (newDuration == newDuration.zero())
```

Comment:
Hmm, I find this style a bit odd. The [`std::chrono::milliseconds::zero()`](https://en.cppreference.com/w/cpp/chrono/duration/zero) method is `static`, but here it's being called through an instance variable.

Reply:
How would you feel about `decltype(newDuration)::zero()`? It allows for flexibility later if, for whatever reason, the duration type changes from milliseconds such that the code doesn't have to change.

Reply:
That had actually come to mind when reviewing the code. That would be acceptable. It's a little bit longer, but clear, and flexible.

Reply:
> 
> 
> That had actually come to mind when reviewing the code. That would be acceptable. It's a little bit longer, but clear, and flexible.

It's actually shorter by 5 characters. :P